packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -133,23 +140,40 @@ export class EditManager<
 		public readonly changeFamily: TChangeFamily,
 		public readonly localSessionId: SessionId,
 		private readonly mintRevisionTag: () => RevisionTag,
+		private readonly logger?: ITelemetryLoggerExt,
```

Comment:
It looks like we don't need to keep the logger instance as a member of this class.

Reply:
```suggestion
		readonly logger?: ITelemetryLoggerExt,
```